dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java

```diff
@@ -695,8 +696,8 @@ public void deleteWorkflowGroupForRole(Context context, HttpServletRequest reque
         throws SQLException, WorkflowConfigurationException, AuthorizeException, WorkflowException, IOException {
         Group group = workflowService.getWorkflowRoleGroup(context, collection, workflowRole, null);
         if (!poolTaskService.findByGroup(context, group).isEmpty()) {
-            throw new UnprocessableEntityException("The Group that was attempted to be deleted " +
-                                                       "still has Pooltasks open");
+            // todo: also handle claimed tasks that would become associated with this group once returned to the pool
```

Comment:
ahah! thanks @ybnd you have obviously spot a future bug here.
Have you already checked how much complex could be to fix it right now? do you mind to create please a ticket for this issue if we need to deal with it separately and change the comment in a FIXME with the reference to the ticket?
Personally I'm not convinced that we should prevent the deletion of a group if there are only tasks already claimed but we will need of course to assure that
- already claimed tasks can be performed
- unclaiming a such task should result in the workflow to proceed as if the group was not defined when the item comes in the first time (i.e. if there are a next step it should go here otherwise proceed until the publication)

Reply:
I'd already opened #8272 for that, I'll update the comment